### DISCUSSION TITLE
Extreme switch id not found

### DISCUSSION THREAD

**Discussion by truongvh96** [prometheus/snmp_exporter#1175] [Q&A]
Cannot adopt OID in EXTREME-VLAN-MIB: extremeVlanAggregationStatus ::= { extremeVlanAggregationEntry 7 }
Cannot adopt OID in EXTREME-VLAN-MIB: extremeVlanOpaqueTaggedPorts ::= { extremeVlanOpaqueEntry 1 }
Cannot adopt OID in EXTREME-VLAN-MIB: extremeVlanOpaqueUntaggedPorts ::= { extremeVlanOpaqueEntry 2 }
Cannot adopt OID in EXTREME-VLAN-MIB: extremeVlanOpaqueControlPorts ::= { extremeVlanOpaqueControlEntry 1 }
Cannot adopt OID in EXTREME-VLAN-MIB: extremeVlanOpaqueControlOperation ::= { extremeVlanOpaqueControlEntry 2 }
Cannot adopt OID in EXTREME-VLAN-MIB: extremeVlanOpaqueControlStatus ::= { extremeVlanOpaqueControlEntry 3 }
Cannot adopt OID in EXTREME-VLAN-MIB: extremeVlanStackTable ::= { extremeVlanStackGroup 1 }
Cannot adopt OID in EXTREME-SERVICES-MIB: extremeRemoteSyslogServerAddressType ::= { extremeRemoteSyslogServerEntry 1 }
Cannot adopt OID in EXTREME-SERVICES-MIB: extremeRemoteSyslogServerAddress ::= { extremeRemoteSyslogServerEntry 2 }
Cannot adopt OID in EXTREME-SERVICES-MIB: extremeRemoteSyslogServerPort ::= { extremeRemoteSyslogServerEntry 3 }
Cannot adopt OID in EXTREME-SERVICES-MIB: extremeRemoteSyslogServerFacility ::= { extremeRemoteSyslogServerEntry 4 }
Cannot adopt OID in EXTREME-SERVICES-MIB: extremeRemoteSyslogServerSeverity ::= { extremeRemoteSyslogServerEntry 5 }
Cannot adopt OID in EXTREME-SERVICES-MIB: extremeRemoteSyslogServerStatus ::= { extremeRemoteSyslogServerEntry 6 }
Cannot adopt OID in EXTREME-VLAN-MIB: extremeVlanStackHigherLayer ::= { extremeVlanStackEntry 1 }
Cannot adopt OID in EXTREME-VLAN-MIB: extremeVlanStackLowerLayer ::= { extremeVlanStackEntry 2 }
Cannot adopt OID in EXTREME-SERVICES-MIB: extremeAuthServerIndex ::= { extremeAuthServerEntry 1 }
Cannot adopt OID in EXTREME-SERVICES-MIB: extremeAuthServerAddressType ::= { extremeAuthServerEntry 2 }
Cannot adopt OID in EXTREME-SERVICES-MIB: extremeAuthServerAddress ::= { extremeAuthServerEntry 3 }
Cannot adopt OID in EXTREME-SERVICES-MIB: extremeAuthServerClientAddressType ::= { extremeAuthServerEntry 4 }
Cannot adopt OID in EXTREME-SERVICES-MIB: extremeAuthServerClientAddress ::= { extremeAuthServerEntry 5 }
Cannot adopt OID in EXTREME-SERVICES-MIB: extremeAuthServerPort ::= { extremeAuthServerEntry 6 }
Cannot adopt OID in EXTREME-SERVICES-MIB: extremeAuthServerSecret ::= { extremeAuthServerEntry 7 }
Cannot adopt OID in EXTREME-SERVICES-MIB: extremeAuthServerReTransmit ::= { extremeAuthServerEntry 8 }
Cannot adopt OID in EXTREME-SERVICES-MIB: extremeAuthServerType ::= { extremeAuthServerEntry 9 }
Cannot adopt OID in EXTREME-SERVICES-MIB: extremeAuthServerIsPrimary ::= { extremeAuthServerEntry 10 }
Cannot adopt OID in EXTREME-SERVICES-MIB: extremeAuthServerAccessType ::= { extremeAuthServerEntry 11 }
Cannot adopt OID in EXTREME-SERVICES-MIB: extremeAuthServerStatus ::= { extremeAuthServerEntry 12 }
Cannot adopt OID in EXTREME-VLAN-MIB: extremeVlanEncapsIfEntry ::= { extremeVlanEncapsIfTable 1 }
Cannot adopt OID in EXTREME-VLAN-MIB: extremeVlanProtocolDefName ::= { extremeVlanProtocolDefEntry 1 }
Cannot adopt OID in EXTREME-VLAN-MIB: extremeVlanProtocolDefDllEncapsType ::= { extremeVlanProtocolDefEntry 2 }
Cannot adopt OID in EXTREME-VLAN-MIB: extremeVlanProtocolDefValue ::= { extremeVlanProtocolDefEntry 3 }
Cannot adopt OID in EXTREME-VLAN-MIB: extremeVlanProtocolDefStatus ::= { extremeVlanProtocolDefEntry 4 }
extremeVlanGroup: Unknown Object Identifier (Sub-id not found: (top) -> extremeVlanGroup)

**Comment by systex-leo** [comment]
Is this problem fixed? I have the same question.

**Reply by bastischubert** [reply]
can you provide a more detailed question, what steps where taken and also used `snmp.yml` or `generator.yml`

**Reply by systex-leo** [reply]
Dear Maintainer,

Today, I restored my virtual environment, re-downloaded the project, and ran `make generate` again to import the MIB-related files. There were no error messages output.

However, the issue last time was due to the parameter `UBNT_DL_URL URL` http://dl.ubnt-ut.com/snmp in the Makefile for importing MIB files, which was inaccessible during those days.

**Reply by bastischubert** [reply]
Are the generated configs working as expected?
